### DISCUSSION TITLE
How can I get a responsive side bar like the Fluent website?

### DISCUSSION THREAD

**Discussion by 15MariamS** [microsoft/fluentui#30238] [Q&A]
I'd love for my sidebar to collapse into the bottom like in the Fluent UI v9 webpage. Anyone know how to do this if I'm using an inline drawer for my sidebar?
<img width="273" alt="Screenshot 2024-01-05 at 1 57 40 PM" src="https://github.com/microsoft/fluentui/assets/81782111/f16a3467-6412-483b-a4fc-b54e6b2e562f">

**Comment by marcosmoura** [comment]
@15MariamS This cannot be achieved by default, but here is a custom example on how to implement a similar solution:
[https://codesandbox.io/p/sandbox/epic-cache-ys84tv](https://codesandbox.io/p/sandbox/epic-cache-ys84tv)
